### PR TITLE
Add support for webpack2 loader options. Fixes #115. Fixes #128.

### LINKED ISSUE #115: Plugin syntax for Webpack v2
```js
var LessPluginCleanCSS = require('less-plugin-clean-css');

module.exports = {
  ...
  lessLoader: {
    lessPlugins: [
      new LessPluginCleanCSS({advanced: true})
    ]
  }
};
```

This code doesn't seem to work in Webpack v2, any idea what the correct syntax should be? Currently I get this error message:
```
throw new WebpackOptionsValidationError(webpackOptionsValidationErrors);
```

### LINKED ISSUE #128: The param 'LessPlugins' can't work
When configing webpack(2.x) and use less-loader like below:
```
{
  test: /\.less$/,
  use: [
    'style-loader',
    { loader: 'css-loader', options: { importLoaders: 1 } },
    {
      loader: 'less-loader',
      options: { LessPlugins: new lessPluginAutoPrefix({ cascade: true, browsers: ["last 5 versions"] }) }
    }
}
```
The plugin will not work unless config it like this below:
```
plugins: [
  new webpack.LoaderOptionsPlugin({
    options: {
        lessLoader: {
           lessPlugins: [
              new lessPluginAutoPrefix({ cascade: true, browsers: ["last 5 versions"] })
           ]
       }
   }
 })
]
```
I read the code in less-loader and find the cause in the index.js
file:less-loder/index.js
line:50
```
// If present, add custom LESS plugins.
if (this.options[configKey]) {
	config.plugins = config.plugins.concat(this.options[configKey].lessPlugins || []);
}
```
and the `configKey`  is default `lessPlugins`

### PR DESCRIPTION
**Pull request by jakwuh**
No tests were added since it will require webpack2. However we can't still move to webpack2 since it is currently in beta.
Tested against initial tests and also locally against support for less-loader webpack2 options param.

### CODE REVIEW

**Comment by goloveychuk** [comment]
hey, webpack2 released. Could you pls review this PR and merge if it's ok?

File: index.js [hunk 1]
```diff
@@ -51,6 +51,11 @@ module.exports = function(source) {
 		config.plugins = config.plugins.concat(this.options[configKey].lessPlugins || []);
 	}
 
+	// Also support webpack 2 loader.options.
+	if (query.lessPlugins) {
```
Comment (sokra) [inline index.js]: Could you name it `plugins`?

Could you refactor the config stuff to use `loaderUtils.getLoaderConfig(this, "lessLoader")`? This function handles the config stuff and merging.

Comment (jakwuh) [inline index.js]: Done

**Comment by jhnns** [comment]
Thank your for your PR 👍, but this has been superseded by https://github.com/webpack/loader-utils/issues/56 and https://github.com/webpack-contrib/less-loader/pull/147/files

**Pull request closed without merging**